src/schemaCompare/schemaCompareWebViewController.ts

```diff
@@ -727,8 +730,65 @@ export class SchemaCompareWebViewController extends ReactWebviewPanelController<
                 }
 
                 this.updateState(state);
+            } else {
+                if (result.blockingDependencies) {
+                    const diffEntry = payload.diffEntry;
+                    const diffEntryName = this.formatEntryName(
+                        diffEntry.sourceValue ? diffEntry.sourceValue : diffEntry.targetValue,
+                    );
+
+                    const firstBlockingDependencyEntry = result.blockingDependencies[0];
```

Comment:
if we know of multiple blocking dependencies, I assume we should present them all at once?